src/index.tsx

```diff
@@ -1,20 +1,27 @@
 import ReactDOM from 'react-dom';
+// import { createRoot } from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { HashRouter as Router } from 'react-router-dom';
 
 import { store } from './app/store';
 import { App } from './App';
-import { UsersProvider } from './components/UsersContext';
+
+// const domNode = document.getElementById('root');
```

Comment:
There are commented lines in the code which are not needed and should be removed for better code readability.